src/main/java/net/sf/jabref/model/cleanup/FieldFormatterCleanups.java

```diff
@@ -0,0 +1,122 @@
+package net.sf.jabref.model.cleanup;
```

Comment:
Ok, at some point we might think of structuring the model package a little more, but for now it is ok, I think.